github/files/drupal7/scripts/sniffers.yml

```diff
@@ -11,13 +11,12 @@
     artifacts_file: commentinfo.md
     webroot: http://localhost
     installation_profile_name: pp
-    installation_profile_path: "profiles/{{ installation_profile_name }}"
```

Comment:
Profile variable should be defined within profile flow settings file.
Also we need to remove it by default from codesniffer scan, because it throws an error of non existent folder